### ISSUE TITLE
[Feature] Integrate into Pipe Dream File Factories for complete generation of all App fundamentals

### ISSUE THREAD

**Issue by ejntaylor** [mtolhuys/laravel-schematics#43]
With [Pipe Dream](https://github.com/pipe-dream/laravel-create) you can sketch out your models and use these file factories to create Laravel App files or Laravel Nova files:

https://github.com/topics/pipe-dream-file-factory

It would be incredible if there was some way to use the power of Schematics to map out the model relationships and complete the remainder of the build using these File Factories.

Perhaps this ticket is just a jump off point for a conversation rather than a fully spec'd ticket. 

Thanks!

**Comment by mtolhuys** [comment]
Yeah, I've got some mails from programmers doing things like this also thinking about a merge in features.

I have a couple ideas for this. One thing I already made a start for is a "laravel-scaffolding" website where you can do all this and download the files for your project. For these features in the package I still have to think about a good UI approach of doing that. Since I'm mainly a backend developer it's a challenge. 😅 

I also think a combination of these features would be really nice! It's actually the #1 reason I've made all this open-source.
